applications/ShapeOptimizationApplication/custom_utilities/damping/damping_utilities.h

```diff
@@ -0,0 +1,396 @@
+// ==============================================================================
+//  KratosShapeOptimizationApplication
+//
+//  License:         BSD License
+//                   license: ShapeOptimizationApplication/license.txt
+//
+//  Main authors:    Baumgärtner Daniel, https://github.com/dbaumgaertner,
+//                   Willer Matthias, https://github.com/matthiaswiller
+//
+// ==============================================================================
+
+#ifndef DAMPING_UTILITIES_H
+#define DAMPING_UTILITIES_H
+
+// ------------------------------------------------------------------------------
+// System includes
+// ------------------------------------------------------------------------------
+#include <iostream>
+#include <string>
+#include <algorithm>
+
+// ------------------------------------------------------------------------------
+// External includes
+// ------------------------------------------------------------------------------
+#include <boost/python.hpp>
+#include <boost/numeric/ublas/matrix.hpp>
+#include <boost/numeric/ublas/vector.hpp>
+#include <boost/numeric/ublas/io.hpp>
+
+// ------------------------------------------------------------------------------
+// Project includes
+// ------------------------------------------------------------------------------
+#include "../../kratos/includes/define.h"
+#include "../../kratos/processes/process.h"
+#include "../../kratos/includes/node.h"
+#include "../../kratos/includes/element.h"
+#include "../../kratos/includes/model_part.h"
+#include "../../kratos/includes/kratos_flags.h"
+#include "../../kratos/spatial_containers/spatial_containers.h"
+#include "../../kratos/utilities/timer.h"
+#include "../../kratos/processes/node_erase_process.h"
+#include "../../kratos/utilities/binbased_fast_point_locator.h"
+#include "../../kratos/utilities/normal_calculation_utils.h"
```

Comment:
In general, try to use absolute includes from the project root if you can please. It tends to help in case files change location. Should be fine if you just:

```C++
#include "includes/define.h"
#include "processes/process.h"
#include "includes/node.h"
// etc
```